features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioGattServer.cpp

```diff
@@ -818,47 +818,6 @@ void GattServer::setPreferredConnectionParams(const ::Gap::ConnectionParams_t& p
     memcpy(generic_access_service.ppcp + 6, &params.connectionSupervisionTimeout, 2);
 }
 
-ble_error_t GattServer::setDeviceName(const uint8_t *deviceName)
```

Comment:
I would keep these functions inside the cordio gatt server, they will be needed once the feature is exposed publicly.